### ISSUE TITLE
[FEATURE] Collaboration Forum

### ISSUE THREAD

**Issue by TonyGravagno** [mvextensions/mvbasic#29]
Please designate a collaboration / discussion hub where we can exchange notes about this project. Q&A's and project planning aren't suitable for GitHub "Issues".
I recommend one or more of the following:
1) https://groups.google.com/d/forum/mvdbms
2) https://mvtalk.slack.com
3) https://gitter.im/mvextensions/mvbasic (doesn't exist yet)

All such media should be documented in repo README.md files.

If I am assigned as a collaborator, I will handle setup, announcements, and management of all of these.

Thanks.

**Comment by itsxallwater** [comment]
@TonyGravagno are you also @MVDBMS-Solutions? Otherwise this is indeed ironic timing and there's some parallel discussion happening under #30.

**Comment by MVDBMS-Solutions** [comment]
Yeah, I'm just spinning up MVDBMS Solutions as a new business. :)

**Comment by TonyGravagno** [comment]
Um, confirmed. ;)

**Comment by itsxallwater** [comment]
Shoot me an email at mikew AT zumasys.com and I'll share a pre-release version of this extension I just generated this afternoon. We're gearing up to finally deploy this thing and I'm a) happy to see more community participation here and b) equally happy to share a copy and have you help test things out, if you'd like.

For the most part it is just name changes away from MVON#, etc. but it does include some of the PRs we've accepted here so far, plus my language server changes in the other open PR.

**Comment by itsxallwater** [comment]
Sorry to co-opt this thread as far as an advertisement on the pre-release installer for this, but I ended up creating a shareable link so check out https://zumasyscorp-my.sharepoint.com/:u:/g/personal/mikew_zumasys_com/EWF-JqocPylBiVZ8R8fJDqgBHt8xNg_NbQHlYLOJ7rLm6g?e=66oGdW if you want the download. To install you need to go into VS Code -> Extensions -> "..." -> Install from VSIX and then pick the file.

@andrewcole50 @JeffTeter you may also be interested since you've made some prior contributions.

**Comment by andrewcole50** [comment]
> 
> 
> Sorry to co-opt this thread as far as an advertisement on the pre-release installer for this, but I ended up creating a shareable link so check out https://zumasyscorp-my.sharepoint.com/:u:/g/personal/mikew_zumasys_com/EWF-JqocPylBiVZ8R8fJDqgBHt8xNg_NbQHlYLOJ7rLm6g?e=66oGdW if you want the download. To install you need to go into VS Code -> Extensions -> "..." -> Install from VSIX and then pick the file.
> 
> @andrewcole50 @JeffTeter you may also be interested since you've made some prior contributions.

I've used it some today and specifically testing jbase snippets. Renamed the default to snippets-stock.json and snippets-jbase.json to snippets.json to try it. This brought something to light in regards to language syntax vs snippets. Referring to the attached image, in the suggestions box I see the snippets below another OPEN that has no info to it. Does that come from the language syntax files? I looked at MvLanguage.json and the OPEN statement has documentation to it. Is there a way to add useful info to them, fix the documentation, or would the use of snippets just be better with the autocompletions?
![OPEN-suggestion](https://user-images.githubusercontent.com/1955734/64556859-4ad53380-d30e-11e9-83c9-7904aebaec66.jpg)

**Comment by itsxallwater** [comment]
Yep, one comes from the language definition and the other comes from the snippet definition. Without changing to incorporate your expanded snippets definition you can see how that plays out using "sub" since the base extension has a language definition for "sub" and also provides a snippet for it.

At the moment I'm not aware of a nice way to merge the two but if I'm being honest, I haven't been able to spend much time into looking at options for it. On the flip side, the reality that adding all of those things to the snippets causes the list to feel duplicated might give us cause to wait on expanding the adoption of the additional snippets for now.

**Comment by TonyGravagno** [comment]
@andrewcole50 wrote:
> I've used it some today and specifically testing jbase snippets.

Completely different topic for a different issue/ticket...

**Comment by andrewcole50** [comment]
Just replying to my mention...

**Comment by itsxallwater** [comment]
@TonyGravagno I've accepted your merge request, #30, so thanks for your patience on that. I also added you to the readme as a contributor. Would you concur this is ticket can be closed now?

**Comment by MVDBMS-Solutions** [comment]
> Would you concur this is ticket can be closed now?

Thank you very much, but not yet.

With regard to the discussion about IM/chat, a Gitter channel hasn't yet been setup for this repo. This can only be done by a maintainer of a repo (possibly only the owner). I recommend that we create that channel, remove that note from the ReadMe.

Why recommended? It's possible that someone who wants to discuss the code in this extension is not registered in the MVDBMS group, and will not want to register there. Similarly not everyone uses Slack. So for someone who does want to discuss this software from a code perspective, and they are not inclined to use other venues, Gitter is the only other medium that is available.

When the above is worked out, I'll post an announcement to the Google Group "MVDBMS" and the Slack team "mvTalk".

Now that I'm established here I'll continue to contribute as MVDBMS-Solutions. With full appreciation for the ref in the ReadMe already, I'm hoping you will change that to @MVDBMS-Solutions, and thaat will be the only ID I use here from now-forward.

**Comment by itsxallwater** [comment]
Thank you sir. Setup https://gitter.im/mvextensions/community, updated the readme for that link and made the contribution change per your request!
